addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -18,6 +18,7 @@ export class LinkPopover extends Component {
         getExternalMetaData: Function,
         getAttachmentMetadata: Function,
         isImage: Boolean,
+        shouldDisplayWandIcon: Boolean,
```

Comment:
Conceptually it is the banner that is the special case. Displaying buttons is the normal one (after this PR, of course.) Therefore I would invert the meaning of this prop and name it `showReplaceTitleBanner`.